evaluation.py

```diff
@@ -146,6 +147,101 @@ def _non_max_supression(self, plain, threshold, windowSize=3):
         under_thresh_indices = plain < threshold
         plain[under_thresh_indices] = 0
         return plain * (plain == maximum_filter(plain, footprint=np.ones((windowSize, windowSize))))
+        img_v_resize = self._vstack_images(heatmap_imgs)
+
+        cv2.imwrite(filename, img_v_resize)
+
+    """
+        Parameters
+        ----------
+        metadata : object
+        should be metadata associated to a single image
+
+        untransformed_x : int
+        x coordinate to
+    """
+    def __undo_x(self, metadata, untransformed_x):
```

Comment:
Double `__` in python means name mangling (the interpreter automatically adds the class name in front). Is this intentional?

```suggestion
    def _undo_x(self, metadata, untransformed_x):
```

Reply:
No that was not my intention, good catch!